core/sections/clause_7_core.adoc

```diff
@@ -1247,7 +1247,7 @@ include::../requirements/core/REQ_job-op.adoc[]
 
 include::../requirements/core/REQ_job-success.adoc[]
 
-include::../recommendations/core/PER_job-success.adoc[]
+NOTE: In the case where a server supports multiple processing APIs (e.g. OGC API - Processes, OpenEO, etc.) the <<profile-parameter,profile mechanism>> may be use by non-OGC clients to interact with the `/jobs` endpoint and request status information in a schema that they recognize.
```

Comment:
Should the specific link be provided, or a reference to 
https://docs.ogc.org/DRAFTS/18-062r3.html#mediatypes be added? 

Section https://docs.ogc.org/DRAFTS/18-062r3.html#con-neg-by-profile does not directly provide a link to these `profile` links, so it looks like user will be looking around quite a lot to figure out the requirement.